index.js

```diff
@@ -333,6 +333,17 @@
 //. <script src="vendor/sanctuary-bundle.js"></script>
 //. ```
 //.
+//. The bundle can also be included via:
+//. ```html
+//. <script src="https://sanctuary.js.org/vendor/sanctuary-bundle.js"></script>
+//. ```
```

Comment:
I suggest not advertising this option, as we should not encourage people to depend on something mutable.